tests/rust/test_manifest.py

```diff
@@ -301,18 +316,99 @@ def _assert_workspace_manifest_eq(py, rs):
     py_signed_and_encrypted = py_wm.dump_sign_and_encrypt(signing_key, secret_key)
     rs_signed_and_encrypted = rs_wm.dump_sign_and_encrypt(signing_key, secret_key)
 
-    py_decrypted_and_verified = _PyWorkspaceManifest.decrypt_verify_and_load(
+    WorkspaceManifest.decrypt_verify_and_load(
         py_signed_and_encrypted, secret_key, signing_key.verify_key, py_wm.author, py_wm.timestamp
     )
-    rs_decrypted_and_verified = WorkspaceManifest.decrypt_verify_and_load(
-        rs_signed_and_encrypted, secret_key, signing_key.verify_key, rs_wm.author, rs_wm.timestamp
+    _PyWorkspaceManifest.decrypt_verify_and_load(
```

Comment:
Changed the comparaison function to check for the types of both objects and realized that `_PyWorkspaceManifest.crypt_verify_and_load()` returns a Rust type, not a Python type. This is not really a problem as it will happen only when testing. I added the comparaison back (but with `==` instead of the function).